spacy/lang/nl/lex_attrs.py

```diff
@@ -3,38 +3,198 @@
 
 from ...attrs import LIKE_NUM
 
+import regex
 
-_num_words = set("""
-nul een één twee drie vier vijf zes zeven acht negen tien elf twaalf dertien
-veertien twintig dertig veertig vijftig zestig zeventig tachtig negentig honderd
-duizend miljoen miljard biljoen biljard triljoen triljard
-""".split())
+VIJF = 'vijf'
+ZES = 'zes'
+ZEVEN = 'zeven'
+ACHT = 'acht'
+NEGEN = 'negen'
+TIEN = 'tien'
+HONDERD = "honderd"
+DUIZEND = "duizend"
 
-_ordinal_words = set("""
-eerste tweede derde vierde vijfde zesde zevende achtste negende tiende elfde
-twaalfde dertiende veertiende twintigste dertigste veertigste vijftigste
-zestigste zeventigste tachtigste negentigste honderdste duizendste miljoenste
-miljardste biljoenste biljardste triljoenste triljardste
-""".split())
+wrong_ordinals_re = regex.compile(r"""(?x)
+.*(?P<wrong>nulste|eende|tweeste|driede|vierste|vijfste|zeste|zesste|achtde|tienste|honderdde|duizendde)
+$""")
+
+hundreds_units_and_tens_thousand_re = regex.compile(r"""(?x)
+      (
+        (?P<hundreds>twee|drie|vier|vijf|zes|zeven|acht|negen)honderd
+        |
+        (?P<hundred>honderd)
+      )*
+      (
+        (?:
+          (?P<units>eene|tweeë|drieë|viere|vijfe|zese|zevene|achte|negene)
+          n
+        )*
```

Comment:
Why does the `eene|...|negene` alternation end with a `*` instead of a `?`
You don't want to repeat "eenen", right?
Also, I think it's actually more readable to add the n to each option.

Reply:
You are right, I'll implement your suggestions. I know a bit about regular expressions, but there is obviously always more to learn.